src/cohortMembers/dto/cohortMembers-report-filter.dto.ts

```
@@ -0,0 +1,34 @@
+import { ApiProperty } from '@nestjs/swagger';
```

ESLint

[error] Replace `'`@nestjs/swagger`'` with `"`@nestjs/swagger`"`

(prettier/prettier)